### DISCUSSION TITLE
restore backup in another project

### DISCUSSION THREAD

**Discussion by brunowilliang** [pocketbase/pocketbase#1279] [Q&A]
Hi guys
I have a project already not implemented on fly.io,
I made a backup of this project and run it here on my localhost, but I would like to get that same data and send it to another project on fly.io.

does anyone know how to do this?

**Comment by ganigeorgiev** [comment]
The restore process should be similar to the backup but with reverse scp arguments:

```sh
# this will register a ssh key with your local agent (if you haven't already)
flyctl ssh issue --agent

# proxies connections to a fly VM through a Wireguard tunnel
flyctl proxy 10022:22

# run in a separate terminal to upload the pb_data directory
scp -r -P 10022 /your/local/path/to/pb_data root@localhost:/pb/pb_data
```

If you are not executing the `flyctl` command in the directory of your new project, you may have to specify `--config` pointing to the fly config.

Another caveat that could occur is that if the other project was already deployed on fly and has a pb_data, the scp may fail since it cannot override existing files. In this case you can upload it first to a different location (eg. `/pb/pb_data_new`) and then ssh in to your vm, delete the old pb_data, and rename the new one back to `pb_data`.
Or you can try with `rsync`: 

```
rsync -avz -p 10022 -e ssh /local/path/to/pb_data root@localhost:/pb/pb_data
```

After uploading the new pb_data, you'll need to restart your fly vm, eg. `flyctl restart [APPNAME]`
